release/release.go

```diff
@@ -802,6 +843,11 @@ func uploadRelease(v version.Version) {
 		)
 	}
 
+	// Make a JSON feed. Since we only have one version right now, we can declare it here and
+	// always append the current version and reference it with feed.Versions[0]
+	var feed jsonFeed
+	feed.Versions = append(feed.Versions, toolsVersion{v.StringWithoutPre(), make([]toolsDownload, 0)})
```

Comment:
suggestion: to avoid the confusion potential bugs associated with indexing into `feed.Versions`, let's just construct the `[]toolsDownload` in the loop and then create a `toolsVersoin` and a `jsonFeed` to contain it _after_ the loop